### DISCUSSION TITLE
Deprecated Calendar

### DISCUSSION THREAD

**Discussion by joelatgrayv** [primefaces/community#2737] [Figma UI Kit] [answered]
I noticed in PrimeOne that the full calendar has a deprecation note. Is there a replacement for it?

<img width="571" alt="Screenshot 2024-09-23 at 2 34 13 PM" src="https://github.com/user-attachments/assets/ced28ccf-4026-454e-a7b3-f22314b4f19e">

**Comment by onursenture** [comment, marked as answer]
Unfortunately, we do not currently have a component that serves as a replacement for FullCalendar. PrimeTek's plans for 2025 include implementing our own advanced custom calendar component under the advanced components category.

**Reply by joelatgrayv** [reply]
Sweet, thanks
